package.json

```diff
@@ -43,9 +43,11 @@
     "express-handlebars": "^5.2.0",
     "helmet": "^4.2.0",
     "lodash": "^4.17.20",
+    "markdown-it": "^12.0.4",
     "memejs": "^1.2.3",
     "mongoose": "^5.10.16",
     "parse-duration": "^0.4.4",
+    "require-extension": "^0.0.4",
```

Comment:
Is this actually used?

Reply:
Used it for a bit initial, then stopped, will remove it